ui/src/app/components/admin/audit-page-component.tsx

```diff
@@ -50,14 +50,8 @@ export interface AuditPageProps {
 
 const UserInput = ({initialAuditSubject, auditSubjectType, getNextAuditPath, buttonLabel, queryText}) => {
   const [auditSubject, setAuditSubject] = useState(initialAuditSubject);
-  const [loadNextSubject, setLoadNextSubject] = useState(false);
   const [downloadSqlFile, setDownloadSqlFile] = useState(false);
-
-  useEffect(() =>  {
-    if (loadNextSubject) {
-      setLoadNextSubject(false);
-    }
-  }, [loadNextSubject]);
+  const [navSubject, setNavSubject] = useNavigation();
```

Comment:
I didn't read this name as a boolean and  was confused. Maybe something like `doNavigate` or `canNavigate`?

Reply:
Changed to `nextSubjectRequested` - is that clearer?